db/scripts/run

```diff
@@ -1,7 +1,11 @@
 #!/usr/bin/env bash
 set -e
 GIT_BRANCH=$(git -C "$PACKAGE_ROOT" symbolic-ref --short HEAD)
-docker run --rm -d --network=host --name packit-db mrcide/packit-db:$GIT_BRANCH
+docker run --rm -d \
+ --network=bridge \
```

Comment:
I changed this to a bridge network as localhost was getting propagated in host computer on Mac. We should create a custom network when we dockerize all apps for communication between containers.